OpenTabletDriver.Configurations/Configurations/XP-Pen/Artist 22HD.json

```
@@ -0,0 +1,37 @@
+{
+  "Name": "Xp-Pen Artist 22HD",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 476.64,
+      "Height": 268.11,
+      "MaxX": 38161.0,
+      "MaxY": 21458.0
+    },
+    "Pen": {
+      "MaxPressure": 2047,
+      "Buttons": {
+        "ButtonCount": 2
+      }
+    },
+    "AuxiliaryButtons": null,
+    "MouseButtons": null,
+    "Touch": null
+  },
+  "DigitizerIdentifiers": [
+    {
+      "VendorID": 21827,
+      "ProductID": 71,
+      "InputReportLength": 8,
+      "OutputReportLength": null,
+      "ReportParser": "OpenTabletDriver.Vendors.XP_Pen.XP_PenReportParser",
```

Upstream `configs` branch just updated and report parser's namespace has changed.
```suggestion
      "ReportParser": "OpenTabletDriver.Configurations.Parsers.XP_Pen.XP_PenReportParser",
```